sdcm/cluster.py

```diff
@@ -1382,7 +1405,7 @@ def detect_disks(self, nvme=True):
         :param nvme: NVMe(True) or SCSI(False) disk
         :return: list of disk names
         """
-        patt = ('nvme0n*', 'nvme0n\w+') if nvme else ('sd[b-z]', 'sd\w+')
+        patt = ('nvme*n*', 'nvme\w+n\w+') if nvme else ('sd[b-z]', 'sd\w+')
```

Comment:
what are patt[0] and patt[1], why do we need them? please separate to different variables, smth. like:
device_pattern, disk_pattern = ('nvme*n*', 'nvme\w+n\w+') if nvme else ('sd[b-z]', 'sd\w+')